src/backend/tools.ts

```diff
@@ -43,7 +48,36 @@ export const DXVK = {
       },
       {
         name: 'dxvk',
-        url: 'https://api.github.com/repos/doitsujin/dxvk/releases/latest',
+        url: () => {
```

Comment:
maybe this function can be moved outside the `const tools = [...` so we would just set url as a string, and then we don't need the `const download_url = typeof tool.url === 'string' ? tool.url : tool.url()` line if it's always a string

Reply:
if you do that you can even do something like `if (download_url) { tools.push({name: 'dxvk', url: dxvkUrl, etc...}) }` so you can handle the case when DXVK cannot be used (we have the FIXME saying we can't handle that case)

Reply:
Simply not downloading DXVK isn't quite enough, the frontend/other parts of the backend would still try to install it (which would then fail, since the folder isn't there)

Reply:
oh I see

Reply:
I think my first comment still applies though, to not have to do the if typeof === 'string' conditional

Reply:
I've moved it into its own function with aebb5fe now. This is definitely clearer, thanks for the suggestion